src/test/java/org/jenkinsci/plugins/workflow/steps/durable_task/ShellStepTest.java

```diff
@@ -237,7 +238,7 @@ public DescriptorImpl() {
     @Test public void deadStep() throws Exception {
         logging.record(DurableTaskStep.class, Level.INFO).record(CpsStepContext.class, Level.INFO).capture(100);
         WorkflowJob p = j.jenkins.createProject(WorkflowJob.class, "p");
-        p.setDefinition(new CpsFlowDefinition("try {node {isUnix() ? sh('sleep infinity') : bat('ping 127.0.0.1 > nul')}} catch (e) {sleep 1; throw e}", true));
+        p.setDefinition(new CpsFlowDefinition("try {node {isUnix() ? sh('sleep 1000000') : bat('ping 127.0.0.1 > nul')}} catch (e) {sleep 1; throw e}", true));
```

Comment:
@jtnord @michaelneale -- Though neither is actually related to the change I'm making, may as well clean that up too when fixing the other test bug that had been lurking in this code.  Done.